docs: init (#911)
revanced/revanced-manager: docs/2_3_updating.md

# 🔄 Updating ReVanced Manager

In order to keep up with the latest features and bug fixes, it is recommended to keep ReVanced Manager up to date.

## 🪜 Updating steps

> 1. Tap on the **Dashboard** tab in the bottom navigation bar

Reviewer: ```suggestion
1. Navigate to the **Dashboard** tab in the bottom navigation bar
```
